### ISSUE TITLE
`webpack` build fails when `lastmod: true` is used

### ISSUE THREAD

**Issue by octionut** [schneidmaster/sitemap-webpack-plugin#33]
I am trying to use SitemapPlugin with the following configuration:

```
const paths = [
  '/',
  '/contact'
];

...

new SitemapPlugin('<domain.com>', paths, {
  changefreq: 'daily',
  lastmod: true // <-- if i remove this, the error goes away
})
```

When I run `webpack --mode production` I get the following error:

```
ERROR in RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at Object.normalizeURL (/home/[redacted]/Projects/[redacted]/node_modules/sitemap/dist/lib/utils.js:343:50)
    at SitemapStream._transform (/home/[redacted]/Projects/[redacted]/node_modules/sitemap/dist/lib/sitemap-stream.js:74:65)
    at SitemapStream.Transform._read (_stream_transform.js:190:10)
    at SitemapStream.Transform._write (_stream_transform.js:178:12)
    at doWrite (_stream_writable.js:415:12)
    at writeOrBuffer (_stream_writable.js:399:5)
    at SitemapStream.Writable.write (_stream_writable.js:299:11)
    at paths.forEach.path (/home/[redacted]/Projects/[redacted]/node_modules/sitemap-webpack-plugin/lib/index.js:136:15)
    at Array.forEach (<anonymous>)
    at SitemapWebpackPlugin.generateSitemap (/home/[redacted]/Projects/[redacted]/node_modules/sitemap-webpack-plugin/lib/index.js:93:11)
    at SitemapWebpackPlugin.generate (/home/[redacted]/Projects/[redacted]/node_modules/sitemap-webpack-plugin/lib/index.js:165:34)
    at compiler.hooks.emit.tapAsync (/home/[redacted]/Projects/[redacted]/node_modules/sitemap-webpack-plugin/lib/index.js:193:31)
    at _next1 (eval at create (/home/[redacted]/Projects/[redacted]/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:8:1)
    at _err1 (eval at create (/home/[redacted]/Projects/[redacted]/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:22:1)
    at emitHtmlPromise.then (/home/[redacted]/Projects/[redacted]/node_modules/html-webpack-plugin/index.js:290:11)
```

**Comment by schneidmaster** [comment]
Hmm. This would imply that the output of the [date helper](https://github.com/schneidmaster/sitemap-webpack-plugin/blob/master/src/date.js) is not a valid date string. What operating system and version of node.js are you running this build on? What value you get if you open the node repl and manually execute the contents of that function?

**Comment by octionut** [comment]
I'm running node v10.19.0 on Ubuntu 20.04.
```
$ node
>   const dateOptions = { day: "2-digit", month: "2-digit", year: "numeric" };
undefined
>   const date = new Date().toLocaleDateString([], dateOptions).split("/");
undefined
>   const year = date.splice(-1)[0];
undefined
>   date.splice(0, 0, year);
[]
>  date.join("-");
'2020-29-06'
> 
```

It seems that the month and day are the other way round. `new Date().toLocaleDateString()` returns `29/06/2020`.
Would it maybe be beneficial to use `new Date().toISOString()` instead? No need for all those `splice`s and `join`s then.

**Comment by schneidmaster** [comment]
Yeah, I think you are right. `src/date.js` is a holdover from when this library manually generated the sitemap instead of leveraging the `sitemap` library.

**Comment by octionut** [comment]
Awesome work! <3

**Comment by schneidmaster** [comment]
Thanks! Published in 0.8.1.
